ecommerce/extensions/api/v2/views/coupons.py

```diff
@@ -444,6 +463,11 @@ def update_coupon_product_data(self, request_data, coupon):
             coupon.attr.note = note
             coupon.save()
 
+        notify_email = request_data.get('notify_email')
+        if notify_email is not None:
```

Comment:
I think this should be just `if notify_email:` otherwise we are getting an error when we add a notification email and then remove it ... thus form will make it "" - empty string and this will be bypassed here and model save does not handle empty string notification email ... and it gives error, making us unable to remove a notification email.

```suggestion
        if notify_email:
```
<img width="1069" alt="screen shot 2019-01-10 at 1 11 12 pm" src="https://user-images.githubusercontent.com/6991154/50954807-42a3ca80-14d9-11e9-9162-22c87bb2244e.png">

Reply:
@mushtaqak I have fixed the issue. Can you please test it again?